spec/requests/api/graphql/queries/submissions_spec.rb

```diff
@@ -355,5 +355,92 @@
         expect(asset_response['imageUrls']).to eq({})
       end
     end
+
+    describe 'sorting' do
+      let!(:submission) { Fabricate :submission, id: 7, created_at: 1.day.ago }
+      let!(:middle_submission) do
+        Fabricate :submission, id: 8, created_at: 3.days.ago
+      end
+      let!(:last_submission) do
+        Fabricate :submission, id: 9, created_at: 2.days.ago
+      end
+
+      let(:query) do
+        <<-GRAPHQL
+        query {
+          submissions(#{query_inputs}) {
+            edges {
+              node {
+                id
+                createdAt
+              }
+            }
+          }
+        }
+        GRAPHQL
+      end
+
+      context 'without a sort column' do
+        let(:query_inputs) { 'sort: null' }
+
+        it 'returns the submissions sorted ascending by the id column' do
```

Comment:
```suggestion
        it 'returns the submissions sorted descending by the id column' do
```

As [indicated in the resolver](https://github.com/artsy/convection/pull/671/files#diff-b89069eeabeafab59e09aa4453c29bedR88), the default is descending by id. The results of this test also show descending by id, so I think this is just a mis-named test.